pkgs/by-name/sh/sha2wordlist/package.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libbsd
+}:
+
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation {
```
Recursive attributes aren't used here